src/modules/passwordRouter.ts

```diff
@@ -0,0 +1,116 @@
+import express from 'express';
+import nodemailer from 'nodemailer';
+import crypto from 'crypto';
+import bcrypt from 'bcrypt';
+import {prisma} from '../../prisma';
+
```

Comment:
remove this file completely. filename, comment style, logic are not legitimate. Please carefully think about What I told you in the lab. Do you think this code clean?
Also, password related function should be under user module